src/third_party/libcxx/BUILD.gn

```diff
@@ -142,38 +235,163 @@ shared_library("c++") {
     "exported/src/stdexcept.cpp",
     "exported/src/string.cpp",
     "exported/src/strstream.cpp",
+    "exported/src/support/runtime/exception_fallback.ipp",
+    "exported/src/support/runtime/exception_glibcxx.ipp",
+    "exported/src/support/runtime/exception_libcxxabi.ipp",
+    "exported/src/support/runtime/exception_libcxxrt.ipp",
+    "exported/src/support/runtime/exception_msvc.ipp",
+    "exported/src/support/runtime/exception_pointer_cxxabi.ipp",
+    "exported/src/support/runtime/exception_pointer_glibcxx.ipp",
+    "exported/src/support/runtime/exception_pointer_msvc.ipp",
+    "exported/src/support/runtime/exception_pointer_unimplemented.ipp",
+    "exported/src/support/runtime/new_handler_fallback.ipp",
     "exported/src/system_error.cpp",
     "exported/src/thread.cpp",
     "exported/src/typeinfo.cpp",
     "exported/src/utility.cpp",
     "exported/src/valarray.cpp",
+    "exported/src/variant.cpp",
+    "exported/src/vector.cpp",
   ]
 
+  if (host_os == "win") {
+    sources += [
+      "exported/include/support/win32/limits_msvc_win32.h",
+      "exported/include/support/win32/locale_win32.h",
+      "exported/src/support/win32/locale_win32.cpp",
+      "exported/src/support/win32/support.cpp",
+    ]
+  }
+
   public = [
+    "exported/include/algorithm",
+    "exported/include/any",
     "exported/include/array",
     "exported/include/atomic",
     "exported/include/bitset",
+    "exported/include/cassert",
+    "exported/include/ccomplex",
+    "exported/include/cctype",
+    "exported/include/cerrno",
+    "exported/include/cfenv",
+    "exported/include/cfloat",
+    "exported/include/chrono",
+    "exported/include/cinttypes",
+    "exported/include/ciso646",
+    "exported/include/climits",
+    "exported/include/clocale",
+    "exported/include/cmath",
+    "exported/include/codecvt",
+    "exported/include/complex",
+    "exported/include/complex.h",
```

Comment:
Are you sure that `.h` files are the part of public interface? Are they copied to public `include` folder when you install libcxx with CMake?